### PR TITLE
add createBreakpoint

### PR DESCRIPTION
**Pull request by jakapatb**
create createBreakpoint()
default laptopL: 1440, laptop: 1024, tablet: 768
and can custom breakpoint
like this 
createBreakpoint({xl:1440, l:768, s:350})

### CODE REVIEW

File: src/useBreakpoint.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+            const setSideScreen = (): void => {
+                  setScreen(window.innerWidth)
+            }
+            setSideScreen()
+            window.addEventListener('resize', setSideScreen)
+            return () => {
+                  window.removeEventListener('resize', setSideScreen)
+            }
+      })
+      const sortedBreakpoints = Object.entries(breakpoints).sort((a, b) => a[1] >= b[1] ? 1 : -1)
```
Comment (streamich) [inline src/useBreakpoint.ts]: Should we sort the breakpoints once and memoize them using `useMemo()`?

File: src/useBreakpoint.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { useEffect, useState } from 'react'
+
+function useBreakpoint(breakpoints: { [name: string]: number } = { laptopL: 1440, laptop: 1024, tablet: 768 }) {
+      const [useScreen, setScreen] = useState(0)
```
Comment (streamich) [inline src/useBreakpoint.ts]: Usually hooks start with `use*`, maybe we can rename `useScreen` to `screen`.

File: src/useBreakpoint.ts [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import { useEffect, useState } from 'react'
+
+function useBreakpoint(breakpoints: { [name: string]: number } = { laptopL: 1440, laptop: 1024, tablet: 768 }) {
```
Comment (streamich) [inline src/useBreakpoint.ts]: What do you think about a version where there would exist a factory function that creates the `useBreakpoint` hook?

```tsx
const useBreakpoint = createBreakpointHook({ laptopL: 1440, laptop: 1024, tablet: 768 });

const Demo = () => {
  const breakpoint = useBreakpoint();
};
```

**Comment by xobotyi** [comment]
Not sure it should use that name. Absolutely unintuitive.
Don't know what name will be more suitable, but surely not this one.

File: src/useBreakpoint.ts (hunk 3 above)
Comment (xobotyi) [reply src/useBreakpoint.ts]: @streamich please, merge https://github.com/streamich/set-harmonic-interval/pull/4 and https://github.com/streamich/set-harmonic-interval/pull/3
Actually it is ASAP fixes, cuz now all IE support is broken.

**Pull request merged**